keyboards/tweetydabird/lbs6/info.json

```
@@ -0,0 +1,74 @@
+{
+	"keyboard_name": "Little Big Scroll 6",
+	"manufacturer": "Tweetys Wild Thinking",
+	"url": "https://lectronz.com/stores/tweetys-wild-thinking",
+	"maintainer": "TweetyDaBird",	
+	
+	"bootloader": "caterina",
+	"processor": "atmega32u4",
```

```suggestion
	"development_board": "promicro",
```

This will set the MCU, and bootloader, as well as the "pin compatible" setting for converters.